lib/src/endpoints/me.dart

```diff
@@ -80,17 +80,25 @@ class Me extends EndpointPaging {
   }
 
   /// Get the current user's top tracks.
-  Future<Iterable<Track>> topTracks() async {
-    final jsonString = await _api._get('$_path/top/tracks');
+  /// @param limit: the maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.
+  /// @param offset: The index of the first item to return. Default: 0 (the first item). Use with limit to get the next set of items.
+  /// @param time_range: Over what time frame the affinities are computed (long_term, medium_term, short_term)
+  /// For more information, see https://developer.spotify.com/documentation/web-api/reference/#/operations/get-users-top-artists-and-tracks
+  Future<Iterable<Track>> topTracks(int limit, int offset, String timeRange) async {
```

Comment:
These parameters are not required by the Web API.
Do you think it would work modifying this to named optional parameters?